code/headers/qik_2s12v10.hpp

```diff
@@ -224,8 +221,7 @@ namespace r2d2::moving_platform {
          * This function uses the raw reading from motor M1 to estimate how
          * much current flows through the motor in milliampere. Note that the
          * value returned by this function can differ from the actual current
-         * by as much as 20% and according to the datasheet the values needs to
-         * be multiplied by 150.
+         * by as much as 20% and according to the datasheet.
```

Comment:
If you mean that the actual current can differ by 20% and that that is mentioned in the datasheet, the "and" between 20% and according is redundant and should be removed. if this is not the case, the complete sentence "Note that the --- according to the datasheet" makes no sense